### PR TITLE
Moving 'Horizontal Pod Autoscaler' to Concepts section

### PR DESCRIPTION
**Pull request by snehalj2009**
https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/ This page talks about HPA concepts and can not be considered as 'Task'. So moving this page to its correction position i.e. to Concepts section. Also, as HPA is controller, it is moved under the Controller section of Concepts

### CODE REVIEW

**Comment by sftim** [comment]
Quick feedback @snehalj2009 - this would need a redirect; see 
https://github.com/kubernetes/website/blob/master/static/_redirects

**Comment by sftim** [comment]
@snehalj2009 a couple more observations
The new path for the page would be `/docs/concepts/workloads/controllers/horizontal-pod-autoscale/`
Did you mean `/docs/concepts/workloads/controllers/horizontal-pod-autoscaler/`?

I recommend searching the English-locale docs for the _old_ path and editing those so that the pages refer to the new URL. Notably, https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/ should mention the new URL and refer to the page as a concept.

File: static/_redirects [hunk 1]
```diff
@@ -376,8 +376,8 @@
 /docs/user-guide/garbage-collection/     /docs/concepts/workloads/controllers/garbage-collection/ 301
 /docs/user-guide/garbage-collector/*     /docs/concepts/workloads/controllers/garbage-collection/ 301
 /docs/user-guide/gpus/     /docs/tasks/manage-gpus/scheduling-gpus/ 301
-/docs/user-guide/horizontal-pod-autoscaler/*     /docs/tasks/run-application/horizontal-pod-autoscale/ 301
-/docs/user-guide/horizontal-pod-autoscaling/     /docs/tasks/run-application/horizontal-pod-autoscale/ 301
+/docs/user-guide/horizontal-pod-autoscaler/*     /docs/concepts/workloads/controllers/horizontal-pod-autoscale/ 301
+/docs/user-guide/horizontal-pod-autoscaling/     /docs/concepts/workloads/controllers/horizontal-pod-autoscale/ 301
```
Comment (tengqm) [inline static/_redirects]: I think we will need redirects from 
`/docs/tasks/run-application/horizontal-pod-autoscale/` (the task page which might be bookmarked by visitors) to the new `/docs/concepts/workloads/controllers/horizontal-pod-autoscale/` URL.

**Comment by bradtopol** [comment]
Hi @sftim since you are already engaged on this I'm going to assign it to you. Thank you for reviewing this!!!
/assign @sftim

**Comment by tengqm** [comment]
@snehalj2009 Please help resolve the conflict?

**Comment by Rajakavitha1** [comment]
@snehalj2009 please resolve the conflicts!!!!

**Comment by sftim** [comment]
@snehalj2009 how's this going?

I like the proposal; there are a few bits of clean-up to cover and then this looks good to merge.

**Comment by sftim** [comment]
@snehalj2009 would you be willing to revise this pull request based on changes to the page in _master_

**Comment by kbarnard10** [comment]
snehalj2009 I'm going to close this for now since there hasn't been any movement in awhile. Please feel free to reopen when you get a chance to address the comments. Thank you!

**Pull request closed without merging**